src/node-lib/getServerConfig.ts

```diff
@@ -143,6 +143,20 @@ const envVars = satisfies<Record<string, EnvVar>>()({
     availableInBrowser: false,
     default: null,
   },
+  muxSigningKey: {
+    value: process.env.MUX_SIGNING_KEY,
+    envName: "MUX_SIGNING_KEY",
+    required: true,
+    availableInBrowser: false,
+    default: null,
+  },
+  muxSigningSecret: {
+    value: process.env.MUX_SIGNING_SECRET,
+    envName: "MUX_SIGNING_SECRET",
```

Comment:
just thinking about how we handle legacy vs new videos - do we need two sets of keys for the different MUX envs? or continue to use the current api for old videos and this one for new? 🤔

Reply:
Good shout, I've added a query param switch `twenty_twenty` to switch between credential sets. Feel free to give it a better name